lib/observe.js

```diff
@@ -3,6 +3,9 @@ var merge = require('viewdb').merge;
 var LegacyObserver = require('viewdb').Observer;
 var Kuery = require('kuery');
 var projectDocument = require('./utils').projectDocument;
+var LoggerFactory = require('slf').LoggerFactory;
+
+const log = LoggerFactory.getLogger('lx:viewdb_persistence_store_mongodb:observer')
```

Comment:
We should remove `lx` from the logger name.

```suggestion
const log = LoggerFactory.getLogger('viewdb:persistence-store-mongodb:observer');
```